docs/user-guide/getting-started.md

```diff
@@ -1,182 +1,188 @@
 ---
-    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
 ---
 # Quick Start Installation Guide
 
 In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
+Different options for installation include:
+* Quick install (recommended!)
+* Install via Minikube
+* Install in CI
+* Manual Install
+
 ## Before You Begin
 
 The Ambassador Edge Stack is designed to run in Kubernetes for production. The most essential requirements are:
 
 * Kubernetes 1.11 or later
 * The `kubectl` command-line tool
+* Edge Control
 
 ## Install the Ambassador Edge Stack
 
-The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally.
+The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally. Or, if you're a Minikube user, `[check out these directions].`
+
+When you install Edge Control to manage your Edge Stack, it will:
+
+* Obtain a TLS certificate
+* Configure automatic TLS and HTTPS
+* Generate a domain name for you to access your Edge Policy Console and complete advanced configuration
```

Comment:
I would actually inverse these 3 steps: Generate a domain is done first, followed by TLS certificates.